tests/org.eclipse.e4.ui.tests.css.swt/src/org/eclipse/e4/ui/tests/css/swt/CTabItemTest.java

```diff
@@ -356,7 +356,6 @@ void testSelectedShowClose2() {
 		}
 	}
 
-	@Disabled("test was commented before bug 443094")
 	@Test
```

Comment:
Let's assume this one is still not fixed, ok? If you want to activate it, you can do it on a separate PR and monitor the tests the following day. 
```suggestion
	@Disabled("test was commented before bug 443094")
	@Test
```